gulpfile.js

```diff
@@ -195,18 +194,20 @@ gulp.task('bower:css', function() {
 
     // Vendor streams - process each library separately
     // -------------------------------------------------------------
-        // normalize-css
+    // normalize-css
     var streamNormalizeCss = bowerFiles
-            .pipe($.filter(filterNormalizeCss));
-
-    // Package streams - group certain libs into single packages
-    // -------------------------------------------------------------
+        .pipe($.filter(filterNormalizeCss));
 
     // Return vendor CSS stream, for all other 3rd-party CSS
-    return $.mergeStream(streamNormalizeCss)
+    return $.streamqueue({ objectMode: true }, streamNormalizeCss)
```

Comment:
Only one stream is being concatenated for now, but setting up streamqueue in case future css libs are required.